app/components/form/WoW/CategorySelectionPopup/index.tsx

```
@@ -0,0 +1,125 @@
+    //   quality === '0' &&
+    //   itemSubClass === -1
+
+    // if (isAllDefault) {
+    //   setError('Please change at least one additional option (expansion, quality, or subcategory)')
+    //   return
+    // }
+
+    onAdd({
+      item_class: itemClass,
+      item_subclass: itemSubClass,
+      expansion_number: parseInt(expansion),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)